semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -194,6 +194,10 @@ class ResourceSemanticConvention(BaseSemanticConvention):
     GROUP_TYPE_NAME = "resource"
 
 
+class ScopeSemanticConvention(BaseSemanticConvention):
+    GROUP_TYPE_NAME = "scope"
```

Comment:
Please also document the new type here: https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/syntax.md#syntax
And add an example for it in a unit test.
Thanks!

Reply:
You can add a test using the same `yaml` and `.md` from your PR. One for the parsing as in https://github.com/open-telemetry/build-tools/blob/f8714186e10f6bb9fe2f92ada2f3d23760ebed46/semantic-conventions/src/tests/semconv/model/test_correct_parse.py#L55 and maybe another for the markdown generation as in https://github.com/open-telemetry/build-tools/blob/f8714186e10f6bb9fe2f92ada2f3d23760ebed46/semantic-conventions/src/tests/semconv/templating/test_markdown.py#L117

Reply:
added the syntax, and the unit test